pkgs/by-name/go/google-guest-configs/fix-paths.patch

```diff
@@ -1,44 +1,60 @@
-diff -ru3 source.old/src/lib/udev/google_nvme_id source.new/src/lib/udev/google_nvme_id
---- source.old/src/lib/udev/google_nvme_id	1970-01-01 03:00:01.000000000 +0300
-+++ source.new/src/lib/udev/google_nvme_id	2022-02-05 13:30:00.986242869 +0300
+diff --git a/src/lib/udev/google_nvme_id b/src/lib/udev/google_nvme_id
+index f0bba67..152e696 100755
+--- a/src/lib/udev/google_nvme_id
++++ b/src/lib/udev/google_nvme_id
 @@ -17,7 +17,7 @@
- # the metadata server
+ # persistent disk) using the disk names reported by the metadata server.
  
  # Locations of the script's dependencies
 -readonly nvme_cli_bin=/usr/sbin/nvme
 +readonly nvme_cli_bin=@nvme@
  
  # Bash regex to parse device paths and controller identification
- readonly NAMESPACE_NUMBER_REGEX="/dev/nvme[[:digit:]]+n([[:digit:]]+).*"
-diff -ru3 source.old/src/lib/udev/rules.d/64-gce-disk-removal.rules source.new/src/lib/udev/rules.d/64-gce-disk-removal.rules
---- source.old/src/lib/udev/rules.d/64-gce-disk-removal.rules	1970-01-01 03:00:01.000000000 +0300
-+++ source.new/src/lib/udev/rules.d/64-gce-disk-removal.rules	2022-02-05 13:27:42.635300567 +0300
-@@ -14,4 +14,4 @@
- #
- # When a disk is removed, unmount any remaining attached volumes.
- 
--ACTION=="remove", SUBSYSTEM=="block", KERNEL=="sd*|vd*|nvme*", RUN+="/bin/sh -c '/bin/umount -fl /dev/$name && /usr/bin/logger -p daemon.warn -s WARNING: hot-removed /dev/$name that was still mounted, data may have been corrupted'"
-+ACTION=="remove", SUBSYSTEM=="block", KERNEL=="sd*|vd*|nvme*", RUN+="@sh@ -c '@umount@ -fl /dev/$name && @logger@ -p daemon.warn -s WARNING: hot-removed /dev/$name that was still mounted, data may have been corrupted'"
```

Comment:
I think we can now remove umount from the subsitute

Reply:
As discussed before, please do provide a reason during review.